docs/manage/fleet/provision/setup.md

```diff
@@ -342,7 +342,7 @@ You can set optional arguments by defining the following environment variables:
 <!-- prettier-ignore -->
 | Argument | Description |
 | -------- | ----------- |
-| `VIAM_JSON_PATH` | The path to the machine credentials <FILE>viam.json</FILE> file to be copied to the machine. The script will also prompt you for this file if not provided. |
+| `VIAM_JSON_PATH` | The path to the machine cloud credentials <FILE>viam.json</FILE> file to be copied to the machine. The script will also prompt you for this file if not provided. |
```

Comment:
```suggestion
| `VIAM_JSON_PATH` | The path to the machine cloud credentials file (<FILE>viam.json</FILE>) to be copied to the machine. The script will also prompt you for this file if not provided. |
```

[opt] put the file path and name in parens, like in the other occurrences below, to increase consistency